dpbench/benchmarks/l2_norm/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache 2.0
+
+from .l2_norm_initialize import initialize
+from .l2_norm_numba_dpex_k import l2_norm as l2_norm_numba_dpex_k
+from .l2_norm_numba_dpex_n import l2_norm as l2_norm_numba_dpex_n
+from .l2_norm_numba_n import l2_norm as l2_norm_numba_n
+from .l2_norm_numba_np import l2_norm as l2_norm_numba_np
+from .l2_norm_numpy import l2_norm as l2_norm_numpy
+from .l2_norm_sycl_native_ext import l2_norm_sycl
+
```

Comment:
Can we add a module docstring explaining what the benchmark does and showing the algorithm that the implementations implement?

We will eventually do it for all benchmarks, but as `l2_norm` is essentially a new benchmark let us start the good practice with this PR.

Reply:
Added.